### DISCUSSION TITLE
Collaboration Meeting

### DISCUSSION THREAD

**Discussion by utkonos** [plyara/plyara#140] [Ideas]
@larsborn @cccs-rs 
Moving the different threads to one place.

Let's do a meetup via some chat system at a time that works for everyone. We can do some collaboration, deconfliction, and planning.

Zoom or Jitsi work or whatever you have. Maybe @cccs-rs already has some resources for that?

**Comment by cccs-rs** [comment]
My organization typically uses Microsoft Teams for setting up meetings but I also have an account for Discord and Slack.

**Comment by larsborn** [comment]
All of the above work for me! Which time zones are you all in? I'm on GMT+1. I'm fine with basically any day of the week. Do you all have any preferences?

Happy New Year! :-)

**Reply by cccs-rs** [reply]
I'm in EST (GMT-5), I think anytime would be during the work week is fine on my side.

Happy New Year! 🥳

**Reply by utkonos** [reply]
EST and @larsborn it looks like your invite failed. I resent it.

we can do an AM meeting so it's good for @larsborn 

how about later in the week next week or the week after?

and we can use https://meet.jit.si/ it's free and pretty easy.

**Reply by utkonos** [reply]
and happy new year!

**Reply by utkonos** [reply]
Mondays are typically not a great day for early meetings. Here is a set of choices for business hours that @cccs-rs and @larsborn timezones overlap in the next two weeks. I can be available at any of these times. Please just check off the ones that work for you, and the one that intersects and is closest to now, we'll use. I'll post a Jitsi link here just before the meeting (just so a link isn't floating around for so long that it invites unwanted guests).

If none of these times/dates work, please post that here too and I will shift all of it two more weeks forward.

# @larsborn 
- [x] Jan 14, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 14, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 14, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 15, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 15, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 15, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 16, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 16, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 16, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [ ] Jan 17, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [ ] Jan 17, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 17, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 21, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 21, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 21, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 22, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 22, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 22, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 23, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [ ] Jan 23, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 23, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [ ] Jan 24, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [ ] Jan 24, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 24, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1

# @cccs-rs  
- [ ] Jan 14, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 14, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 14, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 15, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 15, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 15, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 16, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 16, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 16, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 17, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 17, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 17, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [ ] Jan 21, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 21, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 21, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 22, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 22, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 22, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 23, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 23, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 23, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 24, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 24, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 24, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1

**Reply by larsborn** [reply]
Thanks for creating these lists and making them so easy to use (with all the different timezones on them)!

**Reply by utkonos** [reply]
@larsborn you're welcome! I also thought that I could make time at all those timeslots, but work put meetings on my schedule after I posted this. So, here's mine. Looks like this is the winner:

Jan 15, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1

I will post a Jitsi link here about 30 minutes before the meeting. Hopefully we don't get any uninvited guests, but I there should be a way to kick people and lock the meeting once we're in.

- [ ] Jan 14, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [ ] Jan 14, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [ ] Jan 14, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [ ] Jan 15, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 15, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 15, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 16, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 16, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 16, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 17, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 17, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 17, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 21, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [ ] Jan 21, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [ ] Jan 21, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 22, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 22, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 22, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 23, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 23, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 23, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1
- [x] Jan 24, 08:00-09:00 EST, 13:00-14:00 UTC, 14:00-15:00 GMT+1
- [x] Jan 24, 09:00-10:00 EST, 14:00-15:00 UTC, 15:00-16:00 GMT+1
- [x] Jan 24, 10:00-11:00 EST, 15:00-16:00 UTC, 16:00-17:00 GMT+1

**Reply by utkonos** [reply]
https://meet.jit.si/PowerfulTodaysFitRight

**Reply by cccs-rs** [reply]
Same meeting link?

**Reply by utkonos** [reply]
Good morning. I posted a new one in the new discussion:
https://meet.jit.si/AliveResortsApplaudFlatly
